examples/table/sortable-table.html

```diff
@@ -0,0 +1,227 @@
+              <td role="cell" class="num">7</td>
+            </tr>
+            <tr>
+              <td role="cell">Ralph</td>
+              <td role="cell">Jefferson</td>
+              <td role="cell">Robin, Inc.</td>
+              <td role="cell">456 Main St.</td>
+              <td role="cell" class="num">513</td>
+            </tr>
+            <tr>
+              <td role="cell">Nancy</td>
+              <td role="cell">Jensen</td>
+              <td role="cell">Eagle, Inc.</td>
+              <td role="cell">2203 Logan Dr.</td>
+              <td role="cell" class="num">3.5</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+    <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          HTML table markup is used to create the rows, caption, and to identify the column headers for the data cells.  The use of HTML supports the ARIA principle of using native element semantics whenever possible.<br/>
+          NOTE: The use of the <code>thead</code> and <code>tbody</code> elements to separate the header rows from the sortable data rows helps reduce the complexity of the sorting script.
+        </li>
+        <li>
+          To help screen reader users understand the purpose of the buttons in the column headers, an off-screen description of the sort functionality of the buttons is appended to the caption text.
+          The description is added to the caption instead of to each button to prevent repetitious verbosity that could interfere with understanding of the column titles.
+        </li>
+        <li>To enhance perceivability when operating the sort buttons, visual keyboard focus and hover are styled using CSS <code>:hover</code> and <code>:focus</code> pseudo-classes:
+          <ul>
+            <li>To make it easier to perceive when a button has focus, the focus indicator encompasses both the column label and sort direction icon.</li>
+            <li>The cursor is changed to a pointer when hovering over the button to help people identify it as an interactive element.</li>
+            <li>To make it easier to perceive that clicking either the column label or the sort direction icon will sort the table, hover styles the button and icon in the same way that focus does.</li>
+            <li>To make activating sort easier for people with visual and movement impairments who are using a pointing device, the click target is maximized not only by making both the column label and sort icon clickable but also by using CSS positioning and sizing to make the button fill the entire header cell area.</li>
+          </ul>
+        </li>
+        <li>To ensure the sorting direction icon is synchronized with the value of the <code>aria-sort</code> attribute, CSS selectors based on the <code>aria-sort</code> value are used to render the sort direction icon.</li>
```

Comment:
OK with removing this? This is a coding practice, not an accessibility feature. If it were to be documented anywhere, it should be in a comment in the code.

Reply:
@mcking65 I have removed the reference to using `aria-sort` for controlling the rendering of the visual sort icon